src/displayapp/DisplayApp.h

```diff
@@ -44,8 +44,7 @@ namespace Pinetime {
       enum class States { Idle, Running };
       enum class FullRefreshDirections { None, Up, Down, Left, Right, LeftAnim, RightAnim };
 
-      DisplayApp(Drivers::St7789& lcd,
-                 Components::LittleVgl& lvgl,
+      DisplayApp(Components::LittleVgl& lvgl,
```

Comment:
Shouldn't `Drivers::St7789& lcd` also be removed from https://github.com/JF002/InfiniTime/blob/develop/src/displayapp/DisplayAppRecovery.h#L45 ?

Reply:
Lcd is actually used in DisplayAppRecovery, but it looks like DisplayApp and DisplayAppRecovery must use all the same parameters even if they aren't needed. I would like to improve this as well, but for now I'll just put lcd back in DisplayApp even though it isn't used, just to keep this PR simple.